mep/people/tests/test_views.py

```diff
@@ -662,6 +662,24 @@ def test_list(self):
             msg_prefix='Response has min set twice for inputs'
         )
 
+        # check for max, min and placeholders for birth year
+        # should be 1885 and 1899 respectively
+        self.assertContains(
+            response, 'placeholder="1885"',
+            msg_prefix='Membership widget sets placeholder for max year.'
+        )
+        self.assertContains(
+            response, 'placeholder="1899"',
+            msg_prefix='Membership widget sets placeholder for min year.',
+        )
```

Comment:
Maybe worth a comment why we expect two? It occurred to me a moment of thinking about it (one for each input), but might be worth noting

Reply:
Added!